File: VSCodes/cypher.py
def encrypt(text,s):
    result = ""
 

    for i in range(len(text)):
        char = text[i]
 
        if((ord(char)>=65 and ord(char)<=90) or (ord(char)>=97 and ord(char)<=122)):
          if (char.isupper()):
              result += chr((ord(char) + s-65) % 26 + 65)
  
        
          else:
              result += chr((ord(char) + s - 97) % 26 + 97)
        else:
          result+=text[i]
    return result
 

text = input()
s = (len(text)%20)+3
print ("Text  : " + text)
print ("Shift : " + str(s))
print ("Cipher: " + encrypt(text,26-s))

File: VSCodes/cypher.cpp

#include <bits/stdc++.h>
using namespace std;

 typedef long long int ll;


string encrypt(char text[], ll s,ll n)
{
	string result = "";


	for (ll i=0;i<n;i++)
	{
		if(text[i]==' ')continue;
		if((text[i]-'a'>=0 && text[i]-'z'<=0)|| (text[i]-'A'>=0 && text[i]-'Z'<=0)){
                    // Encrypt Uppercase letters
                if (isupper(text[i]))
                    result += char(int(text[i]+s-65)%26 +65);
        
            // Encrypt Lowercase letters
            else
                result += char(int(text[i]+s-97)%26 +97);
            }

            else {
                result+=text[i];
            }
        }
        

	// Return the resulting string
	return result;
}

// Driver program to test the above function
int main()
{
    ll n=100;
   // cin>>n;
	char text[n];
    cin.getline(text,n);
	ll s = 4;
	cout << "Text : " << text;
	cout << "\nShift: " << s;
	cout << "\nCipher: " << encrypt(text, s,n);
	return 0;
}
